### ISSUE TITLE
[Question]:zero_shot_text_classification运行run_train.py文件时报错AttributeError: 'NoneType' object has no attribute 'backward'

### ISSUE THREAD

**Issue by zhuyuhao2333** [PaddlePaddle/PaddleNLP#6062]
### 请提出你的问题

库版本: paddlenlp==2.5.2
paddlepaddle==2.4.2

数据用的是官方提供的data文件

代码启动命令也是官方提供
python run_train.py  \
    --device cpu \
    --logging_steps 10 \
    --save_steps 100 \
    --eval_steps 100 \
    --seed 1000 \
    --model_name_or_path utc-base \
    --output_dir ./checkpoint/model_best \
    --dataset_path ./data/ \
    --max_seq_length 512  \
    --per_device_train_batch_size 2 \
    --per_device_eval_batch_size 2 \
    --gradient_accumulation_steps 8 \
    --num_train_epochs 20 \
    --learning_rate 1e-5 \
    --do_train \
    --do_eval \
    --do_export \
    --export_model_dir ./checkpoint/model_best \
    --overwrite_output_dir \
    --disable_tqdm True \
    --metric_for_best_model macro_f1 \
    --load_best_model_at_end  True \
    --save_total_limit 1 \
    --save_plm
<img width="700" alt="屏幕截图 2023-05-31 172317" src="https://github.com/PaddlePaddle/PaddleNLP/assets/103302930/39413029-0886-4a56-b680-ac5b170fae8f">

**Comment by LemonNoel** [comment]
1. 检查 `data` 目录下数据是否包括 `train.txt`, `dev.txt`, `test.txt`等文件
2. 将 paddlepaddle 版本升级为 develop 版本，可参考[官网文档](https://www.paddlepaddle.org.cn/)
